### ISSUE TITLE
got unexpected keyword argument 'mythic_encrypts'

### ISSUE THREAD

**Issue by checkymander** [D00MFist/Mystikal#3]
Hey, I'm getting the following error when trying to create an Armed PDF in Mystikal
```
 _______               __   __ __           __
|   |   |.--.--.-----.|  |_|__|  |--.---.-.|  |
|       ||  |  |__ --||   _|  |    <|  _  ||  |
|__|_|__||___  |_____||____|__|__|__|___._||__|
         |_____|

Mystikal: macOS Initial Access Payload Generator
Main Choice: Choose 1 of 8 choices
Choose 1 for Installer Packages
Choose 2 for Mobile Configuration: Chrome Extension
Choose 3 for Mobile Configuration: Webloc File
Choose 4 for Office Macros: VBA
Choose 5 for Office Macros: XLM Macros in SYLK Files
Choose 6 for Disk Images
Choose 7 for Armed PDFs
Choose 8 to exit
Please make a choice: 7
Selected Armed PDF
!!! This module currently downloads a pdf to modify as a default example
[+] Copied Template Folder to './Payloads/PDF_Payload'
[+] Logging into Mythic
[+] Creating new apfell payload
Traceback (most recent call last):
  File "mystikal.py", line 141, in <module>
    main()
  File "mystikal.py", line 48, in main
    pdf()
  File "/Users/rt/Downloads/Mystikal/Modules/PDF.py", line 103, in pdf
    loop.run_until_complete(main())
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.8/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/rt/Downloads/Mystikal/Modules/PDF.py", line 85, in main
    await scripting()
  File "/Users/rt/Downloads/Mystikal/Modules/PDF.py", line 56, in scripting
    resp = await mythic.create_payload(p, all_commands=True, wait_for_build=True)
  File "/Users/rt/Downloads/Mystikal/mythic.py", line 4026, in create_payload
    resp = await self.get_payloadtypes()
  File "/Users/rt/Downloads/Mystikal/mythic.py", line 4132, in get_payloadtypes
    tmp.append(PayloadType(**x))
TypeError: __init__() got an unexpected keyword argument 'mythic_encrypts'
```

Mythic Version: Current `main` branch (2.2)
Mystikal Version: Current `main` branch

Any idea why I might be getting this error? pip says all requirements are satisfied and as far as I can see all the settings in MythicSettings.py are correct.

**Comment by D00MFist** [comment]
There are probably issues with my implementation and the current Mythic branch 2.2 (I tested on 2.1.17). 
I plan to update for 2.2 in the next couple weeks which should hopefully resolve. 
If not I'll dig further into this.

**Comment by D00MFist** [comment]
Should be addressed with latest update for Mythic 2.2
